### ISSUE TITLE
Csv report when generating a report with multiple scenarios is not formatted correctly

### ISSUE THREAD

**Issue by mrBoie** [PragmaticFlow/NBomber#468]
When creating a csv report that is reporting multiple scenarios the formatting is incorrect

All of the scenarios are written out on one single line.
This makes it a little bit hard to parse the data properly

**Comment by mrBoie** [comment]
Believe the issue is here in the csvreport class
![image](https://user-images.githubusercontent.com/4946948/172253960-8e2ed5a4-ea7e-447c-a591-7bd63b838e0f.png)

**Comment by mrBoie** [comment]
Updated it and created a small test for it, if you want i could create a pr for it :)

As is:
[nbomber_report_string_empty.csv](https://github.com/PragmaticFlow/NBomber/files/8848218/nbomber_report_string_empty.csv)

patched:
[nbomber_report_newline.csv](https://github.com/PragmaticFlow/NBomber/files/8848289/nbomber_report_newline.csv)

**Comment by AntyaDev** [comment]
Hi @mrBoie 
Great work!
